utils/ccloud_library.sh

```diff
@@ -1035,6 +1035,8 @@ function ccloud::destroy_ccloud_stack() {
   fi
 
   SERVICE_ACCOUNT_ID=$1
+  ENVIRONMENT=$(ccloud::get_environment_id_from_service_id $SERVICE_ACCOUNT_ID)
+  ccloud environment use $ENVIRONMENT
```

Comment:
@ybyzek I've been worried about the destroy script as well w/ statefulnees of the `ccloud` CLI, so this is a good change.  I'm wondering does the demo fail fast if the `ccloud environment use` fails here so that any further commands do no damange?

Reply:
@rspurgeon to address this, latest commit now fails fast with `|| exit 1`.